util/security.py

```diff
@@ -0,0 +1,36 @@
+
+import os
+import rncryptor
+
+from typing import ByteString
+
+SECRET_KEY = os.getenv("SECRET_KEY")
```

Comment:
should we maybe add a `, "default"` so if someone forgets to add it in the env file / variables we would still have some encryption?